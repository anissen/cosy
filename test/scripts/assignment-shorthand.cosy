mut v = 3
print v

v += 4
print v

v -= 2
print v

v *= 2
print v

v /= 4
print v

v %= 2
print v
