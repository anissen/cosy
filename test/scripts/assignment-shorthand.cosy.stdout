3
7
5
10
2.5
0.5
